deploy/internal/service-mgmt.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     prometheus.io/scrape: "true"
     prometheus.io/scheme: http
     prometheus.io/port: "8080"
+    service.beta.openshift.io/serving-cert-secret-name: noobaa-mgmt-serving-cert
```

Comment:
As @ron1 mentioned, is there a reason not to add the alpha annotation too? 
`service.alpha.openshift.io/serving-cert-secret-name`

Reply:
Only relevant to openshift3. Has no benefits on openshift4 and may cause incompatibility issues. I opted to treat earlier version of openshift the same way as we treat k8s

Reply:
Got it.

Reply:
@guymguym @nb-ohad Most OpenShift-friendly Kubernetes projects declare OCP 3.11+ support and do not "treat earlier version of openShift the same way as they treat k8s". I would expect upstream NooBaa to do likewise unless there is a major technical reason why OCP 3.11 cannot be supported. This is especially relevant since a). the project is RedHat-sponsored, b). the OpenShift 4 migration tool is still not yet GA AFAIK and c). OKD 4 is still several months off AFAIK. It is understandable that the current NooBaa focus is on supporting OCP 4.2. However, I would hope OCP 3.11 remains supported as well since Red Hat will support it into 2022. It would be great if NooBaa CI leveraged existing Red Hat resources to continue support for OCP 3.11.

Reply:
Good point @ron1. 
Are you planning to move to OCP 4 once the migration tool is available?
We would be happy to support it better upstream and will support you as best as we can.
You can help us by opening issues/PRs with suggested fixes.
However, at this point we don't have delivery pipeline for OCP 3.11 so cannot detect regressions without community help.
@tamireran FYI

Reply:
@guymguym Unfortunately, we will be on OCP 3.11 for quite a while since OCP 4 introduces so many incompatible changes all the way down to the OS.